edge-patron/src/main/resources/domain/edge-patron/features/patrons.feature

```diff
@@ -0,0 +1,74 @@
+    * def userId = createUserResponse.createUserRequest.id
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostPolicies')
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4057, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4090, materialTypeName: TestMaterial2, itemBarcode: 3333}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostCheckOut') { itemBarcode: 3333,userBarcode: 2222 }
+
+    Given path 'patron/account/' + userId
+    When method GET
+    Then status 200
+    And match response.totalLoans == 1
+
+  Scenario: Return fees/fines per item for a patron
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 3333,username:  testUser3}
+    * def userId = createUserResponse.createUserRequest.id
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4058, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4091,materialTypeName: TestMaterial3, itemBarcode: 4444}
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostOwnerAndFine') {barcode: 3333,materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4091, itemId: 388830d5-95db-4528-95b6-4ec9d37d4058 }
+
+    Given path 'patron/account/' + userId + '?includeCharges=true'
+    When method GET
+    Then status 200
+    And match response.totalChargesCount == 1
+
+  Scenario: Return requests for a patron
+    * call read('classpath:domain/edge-patron/features/util/initData.feature@PostPolicies')
+    * def status = 'Checked out'
+    * def createItemResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostItem')  { itemId: 388830d5-95db-4528-95b6-4ec9d37d4059, materialTypeId: 388830d5-95db-4528-95b6-4ec9d34d4092,materialTypeName: TestMaterial4, itemBarcode: 5555}
+    * def itemId = createItemResponse.itemEntityRequest.id
+    * def itemBarcode = createItemResponse.itemEntityRequest.barcode
+    * def servicePointId = createItemResponse.servicePointEntityRequest.id
+    * def createUserResponse = call read('classpath:domain/edge-patron/features/util/initData.feature@PostPatronGroupAndUser') { userBarcode: 4444,username:  testUser4}
+    * def requesterId = createUserResponse.createUserRequest.id
+    * def requestEntityRequest = read('classpath:domain/edge-patron/features/samples/request/request-entity-request.json')
+
+    Given path 'circulation' ,'requests'
+    And request requestEntityRequest
+    When method POST
+    Then status 201
+
+    Given path 'patron/account/' + requesterId + '?includeHolds=true'
+    When method GET
+    Then status 200
+    Then match response.totalHolds == 1
```

Comment:
Could it be worth checking that it is the expected request, e.g. with the ID or barcode?

Reply:
Yes , i will add itemId.